addons/mail/static/src/models/attachment_image.js

```diff
@@ -36,6 +57,16 @@ registerModel({
                 this.update({ attachmentDeleteConfirmDialog: insertAndReplace() });
             }
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDownloadButton() {
+            if (!this.attachment || !this.attachmentList) {
+                return false;
```

Comment:
`return clear()` (so that it makes use of `default`).

Reasoning for using `default` rather than explicit `false` is because they do not say the same thing: `false` is an intentional and willing decision to not show the download button. Whereas using `default` is more like "I don't care / it doesn't matter / it doesn't apply / etc.".

Choosing a default is to take into account this "non-choice". We want to change it and keep the same intention of "I don't care / it doesn't matter / it doesn't apply / etc.".